bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/inverter.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fronius"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Powerinverter Thing Type -->
+	<thing-type id="powerinverter">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Fronius Symo Inverter</label>
+		<description>Fronius Symo Power Inverter</description>
+
+		<channels>
+			<channel id="inverterdatachannelpac" typeId="inverter_pac"/>
+			<channel id="inverterdatachanneluac" typeId="inverter_uac"/>
+			<channel id="inverterdatachanneliac" typeId="inverter_iac"/>
+			<channel id="inverterdatachanneludc" typeId="inverter_udc"/>
+			<channel id="inverterdatachannelidc" typeId="inverter_idc"/>
+			<channel id="inverterdatachannelfac" typeId="inverter_fac"/>
+
+			<channel id="inverterdatachanneldayenergy" typeId="inverter_energy_day"/>
+			<channel id="inverterdatachannelyear" typeId="inverter_energy_year"/>
+			<channel id="inverterdatachanneltotal" typeId="inverter_energy_total"/>
+
+			<channel id="inverterdatadevicestatusstatuscode" typeId="inverter_statuscode"/>
+			<channel id="inverterdatadevicestatuserrorcode" typeId="inverter_errorcode"/>
```

Comment:
Now, you make the reverse, a channel type for each channel.
Here for example, `devicestatus_errorcode` looks OK as channel type for inverterdatadevicestatusstatuscode and statecode. No need to create 2 different channel types with exactly the same label and description.
Same for `devicestatus_statuscode`.
Maybe there are other cases, I have not checked in detail.